testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -1046,6 +1046,25 @@ async fn test_validator_set() {
     );
 }
 
+#[tokio::test]
+async fn test_verify_validator_state() {
+    let (_env, op_tool, backend, mut storage) = launch_swarm_with_op_tool_and_backend(4).await;
```

Comment:
nit: to make this test cheaper to run, let's only launch a swarm with 1 node. If we need more, e.g., to check duplicates, we can increase this 😄